source/grid/grid_tools.cc

```diff
@@ -2708,11 +2708,10 @@ namespace GridTools
   minimal_cell_diameter (const Triangulation<dim, spacedim> &triangulation)
   {
     double min_diameter = triangulation.begin_active()->diameter();
```

Comment:
`std::numeric_limits<double>::max()`, just to make sure you never use artifical cells in anything.